File: src/main/java/com/lcaohoanq/nocket/constant/EmailSubject.java
package com.lcaohoanq.nocket.constant;

public class EmailSubject {

    public static String subjectGreeting(String name) {
        return """
            nocket Corporation- Welcome %s, thanks for joining us!
            """.formatted(name);
    }

    public static String subjectRequestUpdateRole(){
        return """
            nocket Corporation- New Request to update role
            """;
    }

    public static String subjectForgotPassword(String name) {
        return """
            nocket Corporation- Hi %s, we're here to help you get the password back!
            """.formatted(name);
    }

    public static String subjectRunningApp() {
        return """
            nocket Corporation- Your app is running, Happy Coding!
            """;
    }

    public static String subjectBlockEmail(String name){
        return """
            nocket Corporation- %s, your account has been blocked!
            """.formatted(name);
    }

}

File: src/main/java/com/lcaohoanq/nocket/constant/EmailSubject.kt
package com.lcaohoanq.nocket.constant

class EmailSubject {
    companion object {
        fun subjectGreeting(name: String?): String {
            return """
                nocket Corporation - Welcome $name, thanks for joining us!
            """.trimIndent()
        }

        fun subjectRequestUpdateRole(): String {
            return """
                nocket Corporation - New Request to update role
            """.trimIndent()
        }

        fun subjectForgotPassword(name: String?): String {
            return """
                nocket Corporation - Hi $name, we're here to help you get the password back!
            """.trimIndent()
        }

        fun subjectRunningApp(): String {
            return """
                nocket Corporation - Your app is running, Happy Coding!
            """.trimIndent()
        }

        fun subjectBlockEmail(name: String?): String {
            return """
                nocket Corporation - $name, your account has been blocked!
            """.trimIndent()
        }
    }
}
